internal/connectors/engine/activities/activity.go

```diff
@@ -296,8 +296,8 @@ func (a Activities) DefinitionSet() temporalworker.DefinitionSet {
 			Func: a.StoragePaymentInitiationAdjustmentsList,
 		}).
 		Append(temporalworker.Definition{
-			Name: "StoragePaymentInitiationsAdjusmentsIfPredicateStore",
-			Func: a.StoragePaymentInitiationsAdjusmentsIfPredicateStore,
+			Name: "StoragePaymentInitiationsAdjustmentsIfPredicateStore",
```

Comment:
@laouji I should probably duplicate this, for backward compatibility no?
We might have some workflow in progress that have the previous spelling of the activity.
I"m not 100% sure it's necessary, I prefer to ask before doing it :)